modular_skyrat/modules/tableflip/code/flipped_table.dm

```diff
@@ -0,0 +1,79 @@
+/obj/structure/flippedtable
+	name = "flipped table"
+	desc = "A flipped table."
+	icon = 'modular_skyrat/modules/tableflip/icons/flipped_tables.dmi'
+	icon_state = "metal-flipped"
+	anchored = TRUE
+	density = TRUE
+	layer = ABOVE_MOB_LAYER
+	opacity = FALSE
+	var/table_type = /obj/structure/table
+
+/obj/structure/flippedtable/CanAllowThrough(atom/movable/mover, turf/target)
+	. = ..()
+	var/attempted_dir = get_dir(loc, target)
+	if(table_type == /obj/structure/table/glass) //Glass table, jolly ranchers pass
+		if(istype(mover) && (mover.pass_flags & PASSGLASS))
+			return TRUE
+	if(istype(mover, /obj/projectile))
```

Comment:
Allow them to pass if they're facing the direction in which it is flipped? It's how you'd use tables for cover and shooting